docs/source/examples.buildcell.ipynb

```diff
@@ -0,0 +1,71 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "be6185f2-8bfa-4499-bf7e-faadc6166383",
+   "metadata": {},
+   "source": [
+    "# Random Structures via buildcell\n",
+    "\n",
+    "In Workflow random structures can be generated via the `wfl.generate.buildcell.run()` routine. \n",
+    "It's functionality builds on [AIRSS](https://airss-docs.github.io/technical-reference/buildcell-manual)’s `buildcell` to constructs sensible random structures.\n",
+    "\n",
+    "The example below illustrates its application for the random generation of aluminum unit cells.\n",
+    "Here, we are aiming for a total of 20 structures and, thus, set `inputs` to an iterable of that length.\n",
+    "Next, we define an `OutputSpec` to handle the output structures that will be generated.\n",
+    "In order to have a proper `buildcell_input` available, we are using the `wfl.generate.buildcell.create_input()` routine in this example\n",
+    "where we pass arguments that characterise the systems we are aiming to generate.\n",
+    "Finally, we set the `buildcell_cmd` appropriately to the `buildcell` executable we use on our machine and run the script\n",
+    "to obtain the desired number of random Al-based unit cells."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4fd84da0-9483-4a01-9aa6-8de6d53aed9d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "from wfl.generate.buildcell import create_input, run\n",
```

Comment:
Do you think it'll be more readable if it's
```
from wfl.generate import buildcell
.
.
outputs = buildcell.run(inputs, outputspec, ...)
```
I'm reconsidering my choice to name the actual operations very nondescript things like `run`.